static/src/stylesheets/layout/new-header/_menu.scss

```diff
@@ -35,8 +38,27 @@
         margin-right: $gutter-xlarge + ($veggie-burger-medium / 2);
     }
 
+    // based on https://css-tricks.com/full-width-containers-limited-width-parents/#article-header-id-6
+    @include mq(desktop) {
+        display: none;
+        left: 50%;
+        margin-left: -50vw;
```

Comment:
We only have [partial support](https://caniuse.com/#search=Viewport%20units) for vw on edge and ie. Have you taken a look to see what happens when this isn't supported?